src/amp-story-embed.js

```diff
@@ -47,20 +57,48 @@ export class AmpStoryEmbed {
       'Missing configuration.'
     );
 
+    /** @private {!Window} */
+    this.win_ = win;
+
+    /** @private {!Object<number, Messaging>} */
+    this.messagingFor_ = {};
+
+    /** @private {!Array<!Element>} */
+    this.iframes_ = [];
```

Comment:
A few thoughts for another PR: I'm almost wondering if we should just store the story URLs instead of the iframes, so you can rotate through 3 or 5 iframe elements. Storing the actual iframes will make it harder/impossible to re-use the iframe elements.

Reply:
Yep, this was one of the approaches I was thinking for my next PRs when enabling multiple stories and I agree we should rotate through a fixed amount of iframes.